src/engine/execute.js

```diff
@@ -558,6 +565,16 @@ const execute = function (sequencer, thread) {
             }
         }
     }
+
+    if (runtime.profiler !== null) {
+        if (blockCached._profiler !== runtime.profiler) {
+            prepareBlockProfiling(runtime.profiler, blockCached);
+        }
+        const end = Math.min(i + 1, length);
+        for (let p = start; p < end; p++) {
+            ops[p]._profilerFrame.count += 1;
```

Comment:
Why not use the increment function here?

Reply:
`frame` supports an argument, normally the block opcode. `increment` does not support an argument.